webapp/src/app/core/modules/openapi/api/activity.serviceInterface.ts

```
@@ -31,14 +30,14 @@ export interface ActivityServiceInterface {
      * 
      * @param login 
      */
-    detectBadPracticesByUser(login: string, extraHttpRequestParams?: any): Observable<Array<PullRequestBadPractice>>;
+    detectBadPracticesByUser(login: string, extraHttpRequestParams?: any): Observable<{}>;
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)